testing/e2e/resources/go-test-service/spec.yaml

```diff
@@ -0,0 +1,64 @@
+build:
```

Comment:
We'll need to rename this to `ocibuilder.yaml` after https://github.com/ocibuilder/ocibuilder/pull/55 is merged

Reply:
done